torch/utils/data/_utils/serialization.py

```diff
@@ -0,0 +1,49 @@
+import pickle
```

Comment:
I assume this file needs to be here to prevent cyclic reference. Otherwise, putting it to https://github.com/pytorch/pytorch/tree/master/torch/utils/data/datapipes/utils makes more sense to me.

If we want to move it into datapipes folder, we may need to completely decouple `IterDataPipe` and `MapDataPipe` from `dataset.py` and maybe put them into `torch/utils/data/datapipes/utils/__init__`

Reply:
Yea, it is there to avoid cyclic reference. We can move it inside `datapipes` folder once `IterDataPipe` and `MapDataPipe` move out of `dataset.py`. 

I will look into that soon.